cmd/ocsp-updater/main.go

```diff
@@ -213,18 +214,31 @@ func (updater *OCSPUpdater) sendPurge(der []byte) {
 		return
 	}
 
-	// Create a GET style OCSP url for each endpoint in cert.OCSPServer (still waiting
-	// on word from Akamai on how to properly purge cached POST requests, for now just
-	// do GET)
+	// Create a GET and special Akamai POST style OCSP url for each endpoint in cert.OCSPServer
 	urls := []string{}
+	reverse := func(b []byte) []byte {
+		for i, j := 0, len(b)-1; i < j; i, j = i+1, j-1 {
+			b[i], b[j] = b[j], b[i]
+		}
+		return b
+	}
 	for _, ocspServer := range cert.OCSPServer {
 		if !strings.HasSuffix(ocspServer, "/") {
 			ocspServer += "/"
 		}
+		// Generate GET url
 		urls = append(
 			urls,
 			fmt.Sprintf("%s%s", ocspServer, url.QueryEscape(base64.StdEncoding.EncodeToString(req))),
 		)
+		// Generate POST url, format is the URL that was POST'd to with a query string with
```

Comment:
Can you link to the docs defining this Akamai API, and also defining the byte order of MD5 so it's clear how to go from an MD5 hash to "the first two uint32s in little endian order?"

Also, needs a unitest. Maybe break out the POST section into its own function to make it easier to test separately?